main.mk

```diff
@@ -12,7 +12,7 @@ include $(INFRA_DIR)/icmk/*/*.mk
 # Macroses
 ########################################################################################################################
 # Makefile Helpers
-SVC = $(shell echo $(@) | $(CUT) -d. -f1 )
+SVC = $(shell echo $(@) | grep $(SLASH) > /dev/null && echo $$(echo $(@) | $(CUT) -d/ -f2 | $(CUT) -d. -f1) || echo $$(echo $(@) | $(CUT) -d. -f1))
```

Comment:
I'd say let's rename `SLASH` to `SLASHSIGN` for clarity please

Reply:
Also, I'd say let's start adding comments above, to clarify what's happening here. Something like:
```
# Get Service name. We're parsing Make task name and extracting SVC. So foo.bar will result to SVC=foo and baz/foo.bar to ...
```

Reply:
Fixed